openapi/components/schemas/request-queues/Listrequestsresponse.yaml

```diff
@@ -0,0 +1,26 @@
+title: Listrequestsresponse
```

Comment:
Would you use PascalCase `ListRequestsResponse` for title and file name instead?